AnkiHubSoftware/ankihub_addon
# feat: Add image upload to bulk suggestions

## Proposed changes
This PR adds image upload support to bulk suggestions operations (change and new note suggestion).

## How to reproduce
1. Upload images to note fields on the browse section of the Anki
2. Select all the notes with the images and then click on the right button of the mouse
3. Select the option "AnkiHub: Bulk suggest notes" and then fill in the rationale of the change input
4. Check on the page "Community Suggestions" of the deck if the images are properly shown
